faq/prooffacehelp.php

```diff
@@ -670,18 +670,12 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 
 <p>
 As an alternate option, the Proofreading interfaces have a tiled character
-picker section. Hovering over a character will show an enlarged view of it
-in the box on the left and a tooltip text will appear describing the
+picker section. Selecting one of the menus from the top row will show an
+array of characters. Hovering over a character will show an enlarged view
+of it in the box on the left and a tooltip text will appear describing the
 character. Clicking on a character will insert it into the text.
 </p>
 
```

Comment:
Good changes. Should we also mention that hovering over the menu tiles will show a description of the content of each menu?

Reply:
Yes, good changes. But I agree that mentioning the tooltip for the tiles would be useful.

Reply:
We could change the tooltips for the letter menu tiles to say something like `A variants` and so on.

Reply:
Yes, also a good idea.

Reply:
Added tooltip mentions and added revised tooltips.

Reply:
There is a typo. hoving for hovering.

Reply:
Good catch - thanks!